modules/govuk/manifests/apps/licencefinder.pp

```diff
@@ -81,4 +81,12 @@
     varname => 'PUBLISHING_API_BEARER_TOKEN',
     value   => $publishing_api_bearer_token,
   }
+
+  if $aws_migration {
```

Comment:
This should be `$::aws_migration`

Reply:
I've typed it a million times, I'm not sure how I didn't do that!